control/control_debug_tools/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.5)
+project(control_debug_tools)
+
+find_package(autoware_cmake REQUIRED)
+autoware_package()
+
+find_package(builtin_interfaces REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+find_package(std_msgs REQUIRED)
```

Comment:
These seem not necessary. Please remove them, and check the build again.

Reply:
removed in c30816d8a398a220f1a6c2db0a552f3875aebca8
built successfully!
![image](https://github.com/autowarefoundation/autoware_tools/assets/80969277/2468a329-0057-4325-9816-342253ddba8b)